docs/advanced-config/README.md

```diff
@@ -226,7 +240,7 @@ not explicitly used in either `format` or `right_format`.
 Note: The right prompt is a single line following the input location. To right align modules above
 the input line in a multi-line prompt, see the [`fill` module](/config/#fill).
 
-`right_format` is currently supported for the following shells: elvish, fish, zsh, xonsh, cmd.
+`right_format` is currently supported for the following shells: pwsh, elvish, fish, zsh, xonsh, cmd.
```

Comment:
Please add the warning I had in my PR regarding the drawbacks of the non-native approach in pwsh.

Reply:
Will do

Reply:
Done